client/app/models/project.js

```diff
@@ -125,6 +126,11 @@ export default class ProjectModel extends Model {
 
   @attr() dcpNoticeddate;
 
+  @computed('formattedNoticedDate')
```

Comment:
This should be 'dcpNoticeddate', the variable used in the get function.

Reply:
noted and updated. thx!